egs/wsj/s5/steps/nnet3/chain2/get_raw_egs.sh

```diff
@@ -130,8 +130,8 @@ latdir=$3
 dir=$4
 
 tree=$chaindir/${lang}.tree
-trans_mdl=$chaindir/init/${lang}.mdl  # contains the transition model and a nnet, but
-                                   # we won't be making use of the nnet part.
+trans_mdl=$chaindir/init/${lang}_trans.mdl  # contains the transition model and a nnet, but
```

Comment:
is the comment still accurate?

Reply:
Ok, I removed this since it doesn't have any sense now